packages/grafana-ui/src/components/ClickOutsideWrapper/ClickOutsideWrapper.tsx

```diff
@@ -26,21 +28,43 @@ export class ClickOutsideWrapper extends PureComponent<Props, State> {
     includeButtonPress: true,
     parent: typeof window !== 'undefined' ? window : null,
     useCapture: false,
+    setHandlers: true,
+    className: null,
   };
   myRef = createRef<HTMLDivElement>();
   state = {
     hasEventListener: false,
   };
 
   componentDidMount() {
+    if (this.props.setHandlers) {
+      this.setHandlers();
+    }
```

Comment:
Not enthusiastic about this change to the API. If you don't want to listen to clicks outside, you should not render this component.

Reply:
Well, but this component can be treated not as "a dumb component", but smart one - "if needed, I can listen to clicks". In case of "a dumb component" you need to put additional logic to upper level or may be create another simple component "div or ClickOutsideWrapper". For me, both approaches can live, no need to choose only one.

Reply:
I agree with @joshhunt that changing the public API of a package used by other systems should be on the bottom of our list of changes needed for a bug fix.
This does not make the component smart, nor is it dumb before it. The only differences is that now it accepts a prop that is flag that is supposed to set it to work or not work. The `ClickOutsideWrapper` has only one job - catch clicks. Passing it a "do-not-do-anything" prop, which `setHandlers={false}` is, is not the best way to go.